imagetest/test_suites/network_perf/targets.txt

```diff
@@ -0,0 +1,154 @@
+{
+        "c3-standard-4"  : 100,
+        "c3-standard-8"  : 100,
+        "c3-standard-22" : 100,
+        "c3-standard-44" : 100,
+        "c3-standard-88" : 100,
+        "c3-standard-176": 100,
+        "c3-highcpu-4"   : 100,
+        "c3-highcpu-8"   : 100,
+        "c3-highcpu-22"  : 100,
+        "c3-highcpu-44"  : 100,
+        "c3-highcpu-88"  : 100,
+        "c3-highcpu-176" : 100,
+        "c3-highmem-4"   : 100,
+        "c3-highmem-8"   : 100,
+        "c3-highmem-22"  : 100,
+        "c3-highmem-44"  : 100,
+        "c3-highmem-88"  : 100,
+        "c3-highmem-176" : 100,
+        "c3-standard-4-lssd"  : 100,
+        "c3-standard-8-lssd"  : 100,
+        "c3-standard-22-lssd" : 100,
+        "c3-standard-44-lssd" : 100,
+        "c3-standard-88-lssd" : 100,
+        "c3-standard-176-lssd": 100,
+
+        "e2-standard-2" : 4,
+        "e2-standard-4" : 8,
+        "e2-standard-8" : 16,
+        "e2-standard-16": 16,
+        "e2-standard-32": 16,
+        "e2-highmem-2" : 4,
+        "e2-highmem-4" : 8,
+        "e2-highmem-8" : 16,
+        "e2-highmem-16": 16,
+        "e2-highcpu-2" : 4,
+        "e2-highcpu-4" : 8,
+        "e2-highcpu-8" : 16,
+        "e2-highcpu-16": 16,
+        "e2-highcpu-32": 16,
+        "e2-micro":  1,
```

Comment:
Let's leave out the shared cored machine types such as e2-micro and e2-small largely because of their memory limitations which will not run some OSes.

Reply:
Done.